config/jest/server/start.mjs

```diff
@@ -30,7 +30,10 @@ export default async function serverStart () {
   try {
     await ready({ timeout })
   } catch (error) {
-    await setup({ command: `PORT=${PORT} node app/start.js`, port: PORT })
+    await setup({
+      command: `cross-env-shell PORT=${PORT} node app/start.js`,
```

Comment:
Previously used `cross-env` but `cross-env-shell` (same package, different executable) ensures our process exit signals work correctly with [`jest-dev-server`](https://www.npmjs.com/package/jest-dev-server)

Reply:
We're using `cross-env-shell` primarily for the **PORT** env to be picked up correctly on Windows

Problem with the suggested comment is that behind the scenes (on Linux, macOS) `SIGINT` and `SIGTERM` signals already worked perfectly. Using `cross-env` would break this where `cross-env-shell` wouldn't

It's also on the [`cross-env-shell` README](https://github.com/kentcdodds/cross-env#cross-env-vs-cross-env-shell)

Think this comment does the trick?

```suggestion
      // Ensure PORT works (on Windows) + SIGINT/SIGTERM signal events
      command: `cross-env-shell PORT=${PORT} node app/start.js`,
```